wix/main.wxs

```diff
@@ -232,6 +232,14 @@
                                 Source='target\$(var.Profile)\nu_plugin_extra_chart_line.exe'
                                 KeyPath='yes'/>
                         </Component>
+                        <Component Id='binary18' Guid='*' Win64='$(var.Win64)'>
+                            <File
+                                Id='exe17'
```

Comment:
this id should be `exe18`

also, down below, in the `ComponentRef` section there needs to be `binary18`

Reply:
Good stuff. Will change soon.